inside_data/lib/src/wordpress/wordpress_search.dart

```diff
@@ -0,0 +1,62 @@
+import 'dart:convert';
+import 'package:inside_data/inside_data.dart';
+import 'package:inside_data/src/wordpress/custom_wordpress_types.dart';
+import 'package:inside_data/src/wordpress/wordpress.dart';
+import 'package:http/http.dart' as http;
+
+
+class WordpressSearch extends Wordpress {
```

Comment:
So that's what the search service essentially does, or would we like to take to it further here as well?

Reply:
You mean app/lib/util/search/search-service.dart?
I think that class isn't needed.

But you're right, making an abstract class InsideSearch for WordpressSearch isn't necessary.
(In data access layer, it's important to keep everything abstracted, for when we want to switch to something else, because every library [sqllite, hivedb, issar, etc] has very different usage. But for search - there's only one method anyway, the whole thing is an abstraction around WordPress, so there's no real maintenance improvements by making an abstract class.)